packages/sui-helpers/packages.js

```diff
@@ -0,0 +1,116 @@
+const path = require('path')
+
+/**
+ * Get absolute paths of packages
+ * @param  {String} cwd
+ * @param  {Array} packages
+ * @return {Array}
+ */
+const getPackagesPaths = (cwd) => {
+  const getPath = path.join.bind(this, cwd)
+  return (packages) => packages.map((pkg) => getPath(pkg))
+}
+
+/**
+ * Get Package JSON info
+ * @param  {String} packagePath Absolute pathç
+ * @return {Object} {} in case of error
+ */
+const getPackageJson = (packagePath) => {
+  try {
+    return require(path.join(packagePath, 'package.json'))
+  } catch (e) {
+    return {}
+  }
+}
+
+/**
+ * Get list of dependencies of a path
+ * @param  {String} packagePath Absolute pathç
+ * @return {Array} Both prod and dev deps are returned
+ */
+const getPackageDependencies = (packagePath) => {
+  const pkg = getPackageJson(packagePath)
+  return [...Object.keys(pkg.dependencies || {}), ...Object.keys(pkg.devDependencies || {})]
+}
+
+/**
+ * Get npm names of packages from paths
+ * @param  {String} cwd
+ * @param  {Array} packages
+ * @return {Array}
+ */
+const getPackagesNames = (cwd) => {
+  const getPaths = getPackagesPaths(cwd)
+  return (packages) => {
+    return getPaths(packages)
+      .map((pkgPath) => getPackageJson(pkgPath).name)
+  }
+}
+
+/**
+ * Get path of a package from npm name
+ * @param  {Array<String>} cwds Absolute path of packages
+ * @param  {Array<String>} packages Names of packages
+ * @param  {String} name Package name
+ * @return {String}
+ */
+const mapNameToPath = (cwds) => (packages) => (name) => cwds[packages.indexOf(name)]
+
+/**
+ * Get which dependencies are included in provided path
+ * @param  {Array<String>} dependencies NPM names of packages
+ * @param  {String} pkgPath Path to browse
+ * @return {Array<Array>} index[0] is a path, index[1] re found dependencies
+ */
+const mapDepedencies = (dependencies) => (pkgPath) => {
```

Comment:
More than map this should be filterDeps or getDependenciesBeingUsed or similar 😜